### PR TITLE
Add initial draft of 2015 kick off email

### PR DESCRIPTION
**Pull request by briandant**

### CODE REVIEW

**Comment by briandant** [comment]
Should probably add a request in here for sponsor for the Feb meetup. Should that be a separate email @johnbaldwin @max-nova?
- Other housekeeping things to talk about?
- Any more transition explanation needed?
- Something to talk about how 2014 ended?
- Talk about how many meetups we plan to have to set expectations?

**Comment by johnbaldwin** [comment]
@briandant 
Just getting caught up on work and emails after massive snow shoveling. I think a single email. In part  because it is all about the February meetup. In part because I like fewer rather than more emails in general. @max-nova ? 

I do like reviewing this on GitHub, makes it much easier. I'll try to review tomorrow over lunch. In general, given that the Django Boston community spans heavy academia to social networking startups and so much in between, I think the best approach in group correspondence is to strike a balance between formality and casualness.

**Comment by johnbaldwin** [comment]
Because of the transition and we need to get our grounding organizing and getting speakers, I'm thinking of bi-monthly to start with, having the goal to go back to monthly as we get our rhythm.

**Comment by briandant** [comment]
> I think the best approach in group correspondence is to strike a balance between formality and casualness.

This one may be more on the casual side :-) 

> Because of the transition and we need to get our grounding organizing and getting speakers, I'm thinking of bi-monthly to start with, having the goal to go back to monthly as we get our rhythm.

Agreed.

File: emails/2015intro.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+*This email is a draft. It hasn't been sent to Django Boston yet*
+Hey hey Django Boston!
+
+It's that time again: time for Django Boston to plan its year, and for all of us to start making awesome happen together.
+
+*Quick housekeeping note: John Baldwin and I (Brian Dant) are stepping in for Max Ulenhuth and John Roth. Thanks John and Max for holding it together over the last year and a half!*
```
Comment (briandant) [inline emails/2015intro.md]: Uhlenhuth > Nova

File: emails/2015intro.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Quick refresher on how Django Boston works:
+- Meetups happen on the last Thursday of the month (always at Akamai in Cambridge)
```
Comment (briandant) [inline emails/2015intro.md]: We've reserved Akamai for the 4th Thursday of every month, not
the last Thursday of every month.

File: emails/2015intro.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+There are a ton of talks for past Djangocons:
+
+Djangocon 2014: https://www.youtube.com/watch?v=cqP758k1BaQ&list=PLE7tQUdRKcybbNiuhLcc3h6WzmZGVBMr3
+Djangocon 2013: https://www.youtube.com/watch?v=drWhGUdadGw&list=PLtqtTJ4wP09YOFqm_lBCoQtmS6S0omW3J
+Djangocon Europe 2014: http://eventifier.com/event/djangoconeuropw/videos?page=2
+Search SpeakerDeck for Django: https://speakerdeck.com/search?utf8=%E2%9C%93&q=django (this brings up 44 pages of talks!)
+
+2. Get an idea / content for a talk from a blog post
+
+Smart people write things on the internet. Grab there stuff (ask first), and turn it into a talk. A good place to start is the Lincoln Loop mailing list archives: http://us2.campaign-archive1.com/home/?u=24fce4628ba1d5814debef334&id=2d6dd01daf
```
Comment (briandant) [inline emails/2015intro.md]: 2nd sentence should start "Grab their stuff....."

File: emails/2015intro.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+But all jokes aside: we've all had great experiences when people share their knowledge, especially when they do it for free. Please take this opportunity to seriously consider helping the community by giving a talk (and making yourself look cool and seem smart, which you are).
+
+Of course, you can give a talk on any topic you want (it should be about Django, or at least something interesting for Django programmers). If you don't know what you want to give a talk on, there are many ideas below. If you don't know what to do, but feel the nudge to share your knowledge, respond to the email, and one of us will help you figure it out. If you don't want to give a talk, but are interested in something, let us know.
+
+Alright, don't even think about closing the inbox, don't defer it, just respond and say "yes! I'll give a talk." 
+
+More ideas on finding a talk:
+
+1. Fork a talk:
```
Comment (briandant) [inline emails/2015intro.md]: @natea: How does this idea sit with you? Useful?

File: emails/2015/feb-meetup-call-for-speakers.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+## Thanks to John Roth and Max Nova
+
+John Baldwin and I (Brian Dant) are stepping in as co-organizers for Max Nova and John Roth. Thanks John and Max for holding it together over the last 18 months or so! (If you have benefited from the contributions from Max and John, please send them a quick note thanking them: uhlenhuthm@gmail.com jroth.solar@gmail.com
+
+## Upcoming Meetup: Feb 26th at Akamai
+
+This month, Shankar Ambady from MassChallenge will be giving a talk on Natural Language Processing using [NLTK](http://www.nltk.org/) and Django. This should be awesome! Shankar has given a talk to Boston Python on NLTK, and now he's adding Django to the mix. RSVP [here](http://www.meetup.com/djangoboston/events/220373782/).
+
+We're still looking for a drink sponsor for the February meetup. If your company is interested in sponsoring, please let us know by responding to this email. Sponsorship will allow us all to go out for drinks together after the meetup, and gives you an opportunity to spread the word about your company and development team.
```
Comment (briandant) [inline emails/2015/feb-meetup-call-for-speakers.md]: @natea: is this enough info for the sponsors? Do we have a page that explains sponsorship?

Comment (natea) [inline emails/2015/feb-meetup-call-for-speakers.md]: @briandant we have a sponsorship page which explains more about how it works: http://www.meetup.com/djangoboston/pages/Sponsoring_a_Django_Boston_Meetup/

So we might want to link to that, which could cut down on the # of questions we receive.

File: emails/2015/feb-meetup-call-for-speakers.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+1. Fork a talk
+
+Many, many talks have been given in the past, so instead of starting from the ground up, you could "fork" somebody else's talk and save a bunch of time, while still bringing lots of benefit to us. With a simple email, it's likely that you could get the permission of the original author to use their slides, code, etc.
+
+Places to find talks:
+
+Djangocon 2014: https://www.youtube.com/watch?v=cqP758k1BaQ&list=PLE7tQUdRKcybbNiuhLcc3h6WzmZGVBMr3
+Djangocon 2013: https://www.youtube.com/watch?v=drWhGUdadGw&list=PLtqtTJ4wP09YOFqm_lBCoQtmS6S0omW3J
+Djangocon Europe 2014: http://eventifier.com/event/djangoconeuropw/videos?page=2
+Search SpeakerDeck for Django: https://speakerdeck.com/search?utf8=%E2%9C%93&q=django (this brings up 44 pages of talks!)
```
Comment (natea) [inline emails/2015/feb-meetup-call-for-speakers.md:49]: Another good place to find talks is http://pyvideo.org which essentially aggregates talks from every Python conference.

File: emails/2015/feb-meetup-call-for-speakers.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+The moral of the story: No matter what happens during your talk, it can't be the worst talk ever, cause I own that. And your talk will likely be THE BEST ONE EVER.
+
+But all jokes aside: we're all relying on the community as we make our living using freely available tools, and free knowledge generously shared by previous speakers. Please seriously consider helping the community by giving a talk (and making yourself look cool and seem smart, which you are!).
+
+Of course, you can give a talk on any topic you want (it should be about Django, or at least something interesting for Django programmers). If you don't know what you want to give a talk on, there are many ideas below. If you don't know what to do, but feel the nudge to share your knowledge, respond to the email, and one of us will help you figure it out. If you don't want to give a talk, but are interested in something, let us know.
```
Comment (natea) [inline emails/2015/feb-meetup-call-for-speakers.md]: Might also want to suggest giving a "lightning talk" which is only 5-10 min and might be a good way to test the waters before committing to a full 45 min talk.

File: emails/2015/feb-meetup-call-for-speakers.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+- Flask vs Django
+- Getting a dev environment set up with Django + Vagrant + Virtualbox
+- Django for a (Rails / PHP / .NET / etc) programmer
+- High Performance Django (a talk that uses the recent book by Lincoln Loop to discuss making Django fast)
+- Testing in Django (Given many times. Never too many)
+- What's up with security in Django? (what do I need to be worried about? What does Django take care of for me)
+- Django Rest Framework
+- The state of real-time applications with Django (what are my options?)
+- Getting the most out of the Django ORM
+
```
Comment (natea) [inline emails/2015/feb-meetup-call-for-speakers.md:74]: Do we want to link to a Google Form that posts to the existing Google Spreadsheet with talk topic ideas and potential speakers?

File: emails/2015/feb-meetup-call-for-speakers.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+If you want to give a talk, but don't think you're ready, or have the full time, email us anyway, and we will try to find somebody more experienced to guide you.
+
+6. Volunteer to help somebody prepare a talk
+
+If you don't have enough time to give a full talk, but have good knowledge in something, let us know, and maybe somebody will do most of the leg-work while you guide them.
+
+## Call for video help
+
+If you have video equipment and skills, we would love to hear from you. You can help us live-stream the talks, and record them to upload to our channels, allowing the speakers contribution to live on FOREVER!
+
```
Comment (natea) [inline emails/2015/feb-meetup-call-for-speakers.md:90]: I've found that you can make pretty decent quality videos if you sit in the front row, where you can see the speaker and the slides, and use something like uStream or Livestream to stream/record the talk with your webcam (might have to swivel your computer around if you have a built-in webcam). The only problem is that the webcams typically don't have very good contrast, so the slides can get washed out if the lightning it not ideal.

Even better would be someone bringing a higher quality camera with zoom capabilities. I've done this approach before but it takes more equipment/wires/power outlets to set up, but the benefit is that the quality is better.

File: emails/2015/feb-meetup-call-for-speakers.md (hunk 6 above)
Comment (briandant) [inline emails/2015/feb-meetup-call-for-speakers.md:49]: Oh yeah! Nice link.

File: emails/2015/feb-meetup-call-for-speakers.md (hunk 9 above)
Comment (briandant) [inline emails/2015/feb-meetup-call-for-speakers.md:90]: Cool. I'll leave this in for now, and we can also announce at the meetup on the 16th

**Pull request merged**